### DISCUSSION TITLE
How students can use github copilot for free？？

### DISCUSSION THREAD

**Discussion by Gaogao105** [community/community#30024] [Copilot Conversations]
How students can use github copilot for free？？No entry found on the github copilot homepage.

**Comment by jfullstackdev** [comment]
not very sure of this one 

https://github.blog/2022-06-21-github-copilot-is-generally-available-to-all-developers/#:~:text=If%20you%27re%20a%20student,GitHub%20Copilot%20at%20no%20cost.

and I'm not very sure whether this will be affected by this 

https://github.com/orgs/community/discussions/29932

**Comment by veverkap** [comment]
> How students can use github copilot for free？？No entry found on the github copilot homepage.

A free subscription for GitHub Copilot is available to verified students, and maintainers of popular open-source repositories on GitHub. If you meet the criteria as an open source maintainer, you will be automatically notified when you visit the GitHub Copilot subscription page. As a student, if you currently receive the GitHub Student Developer Pack, you will also be offered a free subscription when you visit the GitHub Copilot subscription page. For more information about the GitHub Student Developer Pack, see "[Apply to GitHub Global Campus as a student](https://docs.github.com/en/free-pro-team@latest/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student)."

https://docs.github.com/en/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot#github-copilot-pricing
